# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/MediaDecoder.cpp
# Commit: 85ef72ebde43
# Full Hash: 85ef72ebde439c5de258af43faa9bd83a6e23b26
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2019-08-29 21:45:25
# Regressor Bug: 1573102
# File Overlap Count: 1
# Description:
#   Bug 1573102 - Remove mGraph from HTMLMediaElement::OutputMediaStream and OutputStreamManager. r=karlt
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D43666
# ==============================================================================

diff -r f61dd73cac3a -r 85ef72ebde43 dom/media/MediaDecoder.cpp
--- a/dom/media/MediaDecoder.cpp	Thu Aug 29 13:32:31 2019 +0000
+++ b/dom/media/MediaDecoder.cpp	Thu Aug 29 13:32:50 2019 +0000
@@ -239,7 +239,7 @@
 }
 
 void MediaDecoder::AddOutputStream(DOMMediaStream* aStream,
-                                   MediaStreamGraphImpl* aGraph) {
+                                   MediaStreamGraph* aGraph) {
   MOZ_ASSERT(NS_IsMainThread());
   MOZ_ASSERT(mDecoderStateMachine, "Must be called after Load().");
   AbstractThread::AutoEnter context(AbstractMainThread());